app/dao/services_dao.py

```diff
@@ -72,6 +75,93 @@ def dao_count_live_services():
     ).count()
 
 
+def dao_fetch_live_services_data():
+    year_start_date, year_end_date = get_current_financial_year()
+    this_year_ft_billing = FactBilling.query.filter(
```

Comment:
This can just be a filter on the query rather than a subquery

Reply:
but then services without ft_billing records in the dates would not show up in the results at all, right? Or at least this is what happened when I tried to apply it as a filter first...